src/main/resources/db/migration/V1_109__prisoner_balance.sql

```diff
@@ -0,0 +1,13 @@
+create table prisoner_balance_mapping
+(
+    nomis_root_offender_id  bigint                   not null,
+    dps_id                  varchar(36)              not null PRIMARY KEY,
```

Comment:
if this is prison number should it be shorter - I guess this way it could be a `UUID` in future?